src/components/Cards/EventCard/EventCard.tsx

```diff
@@ -134,127 +144,227 @@ const EventCard: React.FC<EventCardProps> = (props) => {
+            </Typography>
+            <div
+              className="link-primary"
+              style={{
+                backgroundColor: theme.colors.primary,
+                color: theme.colors.lightBackground,
+              }}
+            >
+              <a href={"#/profile/" + props.host}>{props.hostName}</a>
+            </div>
+            <p className="card-description"> {props.description}</p>
+            <div className="d-flex align-items-center ">
+              <EventAvailableIcon
+                style={{ fontSize: "26px", color: theme.colors.primary }}
               />
-              <Card.Body>
-                <Card.Title>{props.title}</Card.Title>
-                <Card.Text>
-                  <div className="d-flex align-items-center mb-2">
-                    <i className="bi bi-calendar"></i>{" "}
-                    <span className="ms-2">{props.date}</span>
-                    <i className="bi bi-clock ms-2"></i>{" "}
-                    <span className="ms-2">{props.time}</span>
-                  </div>
-                  Host:{" "}
-                  <a href={"#/profile/" + props.host} className="link-primary">
-                    {props.hostName}
-                  </a>
-                  <br />
-                  {props.description}
-                  <br />
-                  {props.tags.split(",").map((tag, index) => (
-                    <span key={index} className="tag badge me-2">
-                      {tag}
-                    </span>
-                  ))}
-                </Card.Text>
-                {new Date(props.date) > new Date() ? (
-                  <Button
-                    className={`${
-                      (props.isValid && props.isRegistered) || register_data
-                        ? "registration_color"
-                        : "btn-c"
-                    } position-absolute bottom-0 end-0 m-3 `}
-                    onClick={registerForEventX}
-                    disabled={
-                      (props.isValid && props.isRegistered) || register_data
-                    }
-                  >
-                    {(props.isValid && props.isRegistered) || register_data
-                      ? "Registered"
-                      : "Register"}
-                  </Button>
-                ) : (
-                  <Button className="btn-d position-absolute bottom-0 end-0 m-3">
-                    Expired
-                  </Button>
-                )}
-                {/* <Button
+              <span className="card-time">
+                <span className="ms-2">{props.date}</span>
+                <span className="ms-2">
+                  {props.time.replace(/\s\(Indian Standard Time\)/, "")}
+                </span>
+              </span>
+              {new Date(props.date) > new Date() ? (
+                <Button
+                  style={{
+                    color: "black",
+                    textTransform: "none",
+                    fontSize: "16px",
```

Comment:
Update this using `theme.tsx`. Remember that never use `px` for font sizes, user `rem` instead. This helps keeping the webpage responsive.